src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts

```diff
@@ -47,38 +48,13 @@ export const PiaFormSideNavPages = (
   };
 
   const enableReview = (): boolean => {
-    if (
-      pia?.hasAddedPiToDataElements === false &&
-      pia?.isNextStepsSeenForDelegatedFlow === true &&
-      pia?.status !== PiaStatuses.INCOMPLETE &&
-      pia?.status !== PiaStatuses.EDIT_IN_PROGRESS
-    ) {
-      // This is for delegated review
-      if (isMPORole()) {
-        if (!isEditMode) {
-          /* check if the status has review priviliges */
-          return true;
-        } else return false;
-        /* only show in view mode */
-      } else {
-        /* not an MPO user */
-        return false;
-      }
+    let roles;
+    if ('roles' in roleCheck()) {
+      roles = roleCheck().roles[0];
```

Comment:
if we are taking the first index, doesn't that become `role` and not plural roles.